README.md

```diff
@@ -15,13 +14,11 @@
 
 [Live Demo Link](https://livedemo.com)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
```

Comment:
- [ ] Please edit `Getting Started` by adding the instructions on how to set up your project locally.
- [ ] Please remove any unused sections to make your readme file more professional.

![image](https://user-images.githubusercontent.com/68796232/192275322-7afa0f4a-0884-45f8-8390-d3e421e7a9a9.png)

Reply:
Thanks very much